### ISSUE TITLE
How tightly integrated is jmonkey?

### ISSUE THREAD

**Issue by madhephaestus** [NeuronRobotics/NrConsole#26]
I just forked this https://github.com/ThreeCSG/JFXScad and am trying to bring it up for use with the trobot arm. IU am coordinating with Steve Kelly for a standardized code-to-binary pipe so we can incorporate all of the Julia processors. We agreed on CSG as the intermediary, which openscad can do as well. 

If jmonkey can be replaced by the same 3d engine these use, then we can have a single unified Java interface for 3d viewing and modeling and printing. I know it might be a pain at first, but in the end it might be kinda epic. If switching to jfx (the built in 3d engine) is not bad, this might be useful. The support libs also seem to let you load .stls already too.

**Comment by madhephaestus** [comment]
It looks like from Java 1.7 forward the JavaFX 3d engine is built into the jvm, meaning no need to deal with JNI's

**Comment by DrVulpinus** [comment]
I know this is a stupid question, but just to clarify, JFXScad does use the JFX engine, correct?
I should be able to switch to JavaFX without a huge difficulty, especially if it is included in the JVM.

**Comment by madhephaestus** [comment]
Yeah, JavaFX is the backbone of Jfxscad. All JVM 8 and up will have it included
